mapie/regression/quantile_regression.py

```diff
@@ -633,6 +633,8 @@ def predict(
         X: ArrayLike,
         ensemble: bool = False,
         alpha: Optional[Union[float, Iterable[float]]] = None,
+        optimize_beta: bool = False,
```

Comment:
why this change ?

Reply:
Because of function signature: https://stackoverflow.com/questions/51003146/python-3-6-signature-of-method-incompatible-with-super-type-class